src/main/webapp/scripts/my-trips.js

```diff
@@ -184,10 +184,10 @@ async function findHotel() {
   const coords = placesToCoordsWeightArray(locationPlaceObjects);
   const [lat, lng] = centerOfMass(coords);
   const response = await fetch(
-    `https://cors-anywhere.herokuapp.com/https://maps.googleapis.com/maps/api/place/textsearch/json?query=lodging&location=${lat},${lng}&radius=10000&key=${GOOGLE_API_KEY}&output=json`
+    `https://cors-anywhere.herokuapp.com/https://maps.googleapis.com/maps/api/place/textsearch/json?type=lodging&location=${lat},${lng}&radius=10000&key=${GOOGLE_API_KEY}&output=json`
   );
-  const results = await response.json();
-  parseAndRenderHotelResults(results);
+  const { results } = await response.json();
```

Comment:
Does wrapping a variable in curly braces force it into an array?

Reply:
This is called [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment), and makes getting JSON properties much easier. It's a syntactic sugar that makes `results` initialized to `response.json().results`.